iceoryx_hoofs/design/include/iox/detail/newtype/comparable.hpp

```diff
@@ -14,14 +14,12 @@
 // limitations under the License.
 //
 // SPDX-License-Identifier: Apache-2.0
-#ifndef IOX_HOOFS_CXX_NEWTYPE_COMPARABLE_HPP
-#define IOX_HOOFS_CXX_NEWTYPE_COMPARABLE_HPP
-#include "iceoryx_hoofs/internal/cxx/newtype/internal.hpp"
+#ifndef IOX_HOOFS_DESIGN_NEWTYPE_COMPARABLE_HPP
+#define IOX_HOOFS_DESIGN_NEWTYPE_COMPARABLE_HPP
+#include "internal.hpp"
```

Comment:
Should this be the absolute path from the `-I`?

```suggestion
#include "iox/detail/newtype/internal.hpp"
```